tests/test_misclassifications.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import pytest
+
+from presc.misclassifications.misclass_rate import (
+    misclass_rate_feature,
+    show_misclass_rate_feature,
+    show_misclass_rates_features,
+    compute_quantiles,
+    show_quantiles_feature,
+    show_quantiles_features,
+)
+
+
+@pytest.fixture
+def test_dataset():
+    dataset = pd.DataFrame(
+        {
+            "Feature 1": [1., 2.5, 3.5, 5.],
+            "Feature 2": [1., 6., 8., 9.],
+            "Class": ["True", "True", "False", "False"]
+        },
+        columns=["Feature 1", "Feature 2", "Class"]
+    )
+    return dataset
+
+
+@pytest.fixture
+def test_dataset_predictions():
+    dataset_predictions = ["False", "False", "False", "True"]
+    return dataset_predictions
+
+
+def test_misclass_rate_feature(test_dataset, test_dataset_predictions):
+    bin_number = 2
+    result = misclass_rate_feature(
+        test_dataset,
+        test_dataset_predictions,
+        feature="Feature 1",
+        bins=bin_number
+    )
+    assert len(result) == 3
+    assert len(result[0]) == len(result[1]) + 1
+    assert len(result[1]) == bin_number
+
+    assert max(result[1]) <= 1.0
+    assert min(result[1]) >= 0.0
+
+
+def test_show_misclass_rate_feature(test_dataset, test_dataset_predictions):
+    show_misclass_rate_feature(test_dataset, test_dataset_predictions, "Feature 2")
```

Comment:
This is good - but when I run pytest this is opening a graphics device window and showing a plot, and it blocks the test from continuing until the window is closed. This needs to be handled in such a way that the plot isn't acutally displayed (not sure how offhand, probably some pyplot setting).